ASAP-Sports/screens/Homescreen.js

```diff
@@ -3,9 +3,55 @@ import { StyleSheet, Text, View, Button, Image, ScrollView, AsyncStorage, FlatLi
 import AwesomeButton from 'react-native-really-awesome-button';
 import { APP_BASE_URL, COLORS, ASAPStyles } from '../const';
 import GameCard from '../assets/components/GameCard';
+import { Permissions, Notifications } from 'expo';
 
 // NOTIDEAL: Mixing different types in the list is ugly AF.
 
+const PUSH_ENDPOINT = 'http://d8becf45.ngrok.io/users/push-token';
```

Comment:
Don't do this. There is a constant for the apps base URL called APP_BASE_URL. Use it like APP_BASE_URL + '/users/push-token' to get the endpoint URL

Reply:
ohh yea my bad i was gonna add that in similar to how kevin did it in review details this was just for testing